src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -237,13 +214,14 @@ void ChaseMovementGenerator<Creature>::MovementInform(Creature* unit)
 
 //-----------------------------------------------//
 template<>
-bool FollowMovementGenerator<Creature>::EnableWalking() const
+bool FollowMovementGenerator<Creature>::EnableWalking(Creature* owner) const
 {
-    return i_target.isValid() && i_target->IsWalking();
+    return i_target.isValid() && (i_target->IsWalking() || i_target->GetTypeId() == TYPEID_UNIT)
```

Comment:
Was a hack for allowing creature pets to walk. Has been removed and the problem fixed by commit (https://github.com/gashole/TrinityCore/commit/c34f2d0b17df8e3e84a621f44cf92f3e967fcbbe).